sprint0/sprint0/RoomLoader/RoomEnemies.cs

```diff
@@ -37,8 +51,12 @@ public void RegisterEnemies(IEnumerable<IEnemy> unregEnemies)
 
         public void RemoveDead()
         {
-            foreach (IEnemy enemy in enemiesToDie) { 
-                enemies.Remove(enemy);}
+            foreach (IEnemy enemy in enemiesToDie) {
+                if ((int)enemy.Type != 5)
```

Comment:
why cast to int instead of just using the actual enum name? It's less clear with just the number